internal/daysteps/daysteps.go

```diff
@@ -12,9 +19,49 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	vals := strings.Split(data, ",")
+	if len(vals) != 2 {
+		err := errors.New("parsePackage: wrong len data")
+		return 0, 0, err
+	}
+
+	steps, err := strconv.Atoi(vals[0])
+	if err != nil {
+		return 0, 0, errors.New("parsePackage: steps")
+	}
+
+	dur, err := time.ParseDuration(vals[1])
+	if err != nil {
+		return 0, 0, errors.New("parsePackage: duration")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```